### PR TITLE
Adding support for standalone diffs of images 

### PR DESCRIPTION
**Pull request by Damans227**
This PR intends to resolve: https://github.com/jupyterlab/jupyterlab-git/issues/664 

*Note: For now, this is just a draft pr raised for collaboration. Therefore, it's not ready to be merged yet.*

### CODE REVIEW

**Review by Damans227 — COMMENTED** [review]
@fcollonval Hi Frederic, What should be the name of the image diff that gets passed here? Is this value arbitrary?

File: src/index.ts [hunk 1]
```diff
@@ -218,5 +219,12 @@ async function activate(
       createNotebookDiff(model, renderMime, toolbar, translator)
   );
 
+  gitExtension.registerDiffProvider(
+    'ImageDiff',
```
Comment (Damans227) [inline src/index.ts:223]: @fcollonval What should be the name of the image diff that gets passed here? Is this value arbitrary?

Comment (fcollonval) [reply src/index.ts:223]: Yes

**Comment by fcollonval** [comment]
Closed by #1223

**Pull request closed without merging**